### PR TITLE
docs: Show how to get auto-detected Host firewall interfaces

### PR DESCRIPTION
**Pull request by sathieu**
Please ensure your pull request adheres to the following guidelines:

- [ ] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [ ] All code is covered by unit and/or runtime tests where feasible.
- [ ] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] If your commit description contains a `Fixes: <commit-id>` tag, then
      please add the commit author[s] as reviewer[s] to this issue.
- [ ] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [ ] Provide a title or release-note blurb suitable for the release notes.
- [ ] Are you a user of Cilium? Please add yourself to the [Users doc](https://github.com/cilium/cilium/blob/main/USERS.md)
- [ ] Thanks for contributing!

Fixes: #issue-number

*no release notes*

### CODE REVIEW

**Review by aanm — APPROVED** [review]

File: Documentation/security/host-firewall.rst [hunk 1]
```diff
@@ -35,7 +35,8 @@ Deploy Cilium release via Helm:
 
 The ``devices`` flag refers to the network devices Cilium is configured on,
 such as ``eth0``. If you omit this option, Cilium auto-detects what interfaces
-the host firewall applies to.
+the host firewall applies to. The resulting interfaces are shown with the
+``cilium-dbg status`` command below.
```
Comment (learnitall) [inline Documentation/security/host-firewall.rst]: Can we add a new code block to show the output here too? Referencing the output in a different section can be confusing and difficult to follow:

```suggestion
the host firewall applies to. The resulting interfaces are shown in the
output of the ``cilium-dbg status`` command:

.. code-block: shell-session

    $ kubectl exec -n kube-system ds/cilium -- \
         cilium-dbg status | grep ' Host firewall'
```

**Review by learnitall — CHANGES_REQUESTED** [review]
Thank you! One suggestion for you, but then we'll be good to go.

**Comment by sathieu** [comment]
@learnitall I applied you suggestion. Thanks!

File: Documentation/security/host-firewall.rst [hunk 2]
```diff
@@ -35,7 +35,13 @@ Deploy Cilium release via Helm:
 
 The ``devices`` flag refers to the network devices Cilium is configured on,
 such as ``eth0``. If you omit this option, Cilium auto-detects what interfaces
-the host firewall applies to.
+the host firewall applies to. The resulting interfaces are shown in the
+output of the ``cilium-dbg status`` command:
+
+.. code-block: shell-session
```
Comment (learnitall) [inline Documentation/security/host-firewall.rst]: I had a typo in my earlier suggestion, sorry about that!

```suggestion
.. code-block:: shell-session
```

**Review by learnitall — CHANGES_REQUESTED** [review]

File: Documentation/security/host-firewall.rst (hunk 2 above)
Comment (sathieu) [reply Documentation/security/host-firewall.rst]: @learnitall fixed. Thanks!

**Review by learnitall — APPROVED** [review]
Thank you!

**Comment by learnitall** [comment]
CI is green and all reviews are in, but the read-to-merge label wasn't added. Manually adding it now.

**Pull request merged**